greet.spec.js

```diff
@@ -0,0 +1,8 @@
+import { greet } from "./greet";
+
+describe("test for the greet function", () => {
```

Comment:
Missing the test that tests when provider another name than the coaches?

Reply:
done